packages/databricks-vscode/src/extension.ts

```diff
@@ -594,9 +596,7 @@ export async function activate(
         })
     );
 
-    connectionManager.init().catch((e) => {
-        window.showErrorMessage(e);
-    });
+    await confgiModel.init();
```

Comment:
I like the more explicit `connectionManager.init` better. 

Here we rely on target change event for the actual initialisation, which is not obvious. Technically target may not even change between restarts, it's just an internal implementation detail of the ConfigModel/CachedValue logic that we emit those change events on every startup.

Reply:
I think we already have problems because of emitting change events on every startup - for example connection manager calls `updateSyncDestinationMapper` when `workspaceFsPath` changes in the model. But since it "changes" on the startup (and before the connection manager has a chase to do the login), the initial call to `updateSyncDestinationMapper` fails with user facing "Invalid sync destination" error here:
https://github.com/databricks/databricks-vscode/blob/bundle-integ/packages/databricks-vscode/src/sync/SyncDestination.ts#L62

It might be better to avoid emitting change events on startup and do more explicit initialisation.

Reply:
How about we move the listeners to init function? That way, the listeners are registered after login has been attempted once.